src/_angular/font-setting/dtv-font-setting.js

```diff
@@ -383,6 +410,51 @@
               sheet.addRule("@font-face", rule);
             }
           }
+
+          function addVerticalAlignButtons(editor) {
+            editor.addButton("aligntop", {
+              image: "//s3.amazonaws.com/Rise-Images/Icons/align-top.svg",
```

Comment:
@donnapep They could, but then even when a Widget has this repo as a dependency, the Widget repo gulp file will still require moving the images into its dist/ folder when it builds. Seemed like an unnecessary extra step to me and easier to just host in one spot. What you think?